src/styles/navbar.css

```diff
@@ -19,3 +41,26 @@ header {
   align-items: center;
   gap: 30px;
 }
+
+.list-text {
+  display: flex;
+  width: 3.688rem;
+  height: 1rem;
+  font-family: "Roboto", sans-serif;
+  font-size: 1.1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  text-decoration: none;
+  color: #121212;
+}
```

Comment:
- [ ] You have done really well so far, :clap:, however, according to the design requirement, `Home` and `Categories`  texts styles are a bit diferent, The `Categories` text has an opacity of 0.5 while `Home` does not have any opacity. Matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, please pay attention to the red markings on your design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![StudentNav](https://github.com/GhausAliKhan/book-store/assets/43528594/121d4a73-25e4-4005-b7a6-9ba4c552c111)

</td>
      <td>

![epNav](https://github.com/GhausAliKhan/book-store/assets/43528594/aa17a88c-dbf8-4129-9895-00c6e43bc41c)


</td>
    </tr>
  </table>

Reply:
Thank you for your review, I've added the changes 👍🏻

Reply:
You are most welcome 🙂